Add more description for firmware update. (#125)
riscv-non-isa/riscv-ap-tee-io: src/08-attestation.adoc

while post-update reattestation can only unbind TDI after runtime update. That
can remove the gap between update time and attestation time.

> **Note**

> > In this mode, the device will perform updated anyway, no matter the TSM/TVM
> accept the new firmware update or not. If TSM or TVM cannot accept the new

Reviewer: ```suggestion
> In this mode, the device will be updated, regardless of the TSM or the TVM accepting the firmware
update or not. If TSM or TVM cannot accept the new
```

> firmware, the TSM shall terminate the SPDM session with `END_SESSION`,

Reviewer: Either the TSM terminates the SPDM session or the TVM unbinds from the TDI?

Author: Yes, that is better way to keep SPDM session. Just unbind TDI should be enough.

but not acknowledge the event with `EVENT_ACK`.

> 

Reviewer: We have not defined `EVENT_ACK` in the previous section. Should we add a few line to describe it there?

Author: We can say "SPDM event", and SPDM message `EVENT_ACK`
